components/adv-post-form.js

```diff
@@ -93,16 +110,16 @@ export default function AdvPostForm ({ children }) {
               )
             }}
           </VariableInput>
-          {me && router.query.type === 'discussion' &&
+          {me && itemType &&
```

Comment:
Unfortunately, this dynamic info does not work for edits since for edits, the URL is just `/items/:id`.

I think you need to pass the item and determine the type from that.

Reply:
Now item type is determined by router.query.type if available otherwise it will determine from the item fields
3e031af47b7771652c95e24f9dd04c6b81f35e9a